Update CONTRIBUTING.md file (#2709)
vmware/govmomi: CONTRIBUTING.md

$ git commit -m "govc: Add feature ..." -m "Closes: #<issue-number>"
$ git push $USER issue-<number>
```
**Note**:  

> To register the new `govc` command package, import package into `govmomi/govc/main.go` file.

Reviewer: ```suggestion
To register the new `govc` command package, add a blank `_` import to `govmomi/govc/main.go`.
```
